dashboard/config/honeybadger.yml

```diff
@@ -1,5 +1,5 @@
 ---
-api_key: <%=CDO.dashboard_honeybadger_api_key%>
+api_key: "<%=CDO.dashboard_honeybadger_api_key%>"
```

Comment:
In dev/test this is set to the string "000000", but without explicit quotes in YAML, that was turning into the int 0. This was causing a crash in Honeybadger.notify() where api_key `=~`-ed. Apparently this worked in Ruby 3.0, but in Ruby 3.3 its a no method error.